rps_docker_entrypoint.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+die() {
+    echo "ERROR:" "$@" 1>&2
+    if [[ ${NOEXIT} -gt 0 ]]; then
+        echo "Not dying due to NOEXIT.  Feel free to poke around container."
+        sleep inf
+    fi
+    exit 1
+}
+
+create_certificates() {
+    local HOST="${SSL_HOSTNAME:-localhost}"
+    local DAYS=10000
+    local LOG="config/openssl.err"
+    truncate -s 0 "${LOG}"
+
+    # 1. Make fake certificate authority (CA) certificate.  OpenSearch
+    # requires a root certificate to be specified.
+    if [[ (! -f config/cakey.pem) || (! -f config/cacert.pem) ]]; then
+        openssl req -batch -x509 -newkey rsa:4096 -days "${DAYS}" \
+        -subj "/C=US/ST=California/O=Aryn.ai/CN=Fake CA" \
+        -extensions v3_ca -noenc -keyout config/cakey.pem -out config/cacert.pem \
+        2>> "${LOG}" || die "Failed to create CA certificate"
+        echo "Created CA certificate"
+    fi
+
+    # 2a. Create certificate signing request (CSR) for the node certificate.
+    if [[ (! -f config/rps-key.pem) || (! -f config/rps-cert.pem) ]]; then
+        openssl req -batch -newkey rsa:4096 \
+        -subj "/C=US/ST=California/O=Aryn.ai/CN=${HOST}" \
+        -extensions v3_req -addext "basicConstraints=critical,CA:FALSE" \
+        -addext "subjectAltName=DNS:${HOST}" \
+        -noenc -keyout config/rps-key.pem -out config/rps-req.pem \
+        2>> "${LOG}" || die "Failed to create RPS CSR"
+
+        # 2b. Use the fake CA to sign the node CSR, yielding a certificate.
+        openssl x509 -req -CA config/cacert.pem -CAkey config/cakey.pem \
+        -copy_extensions copy -days "${DAYS}" \
+        -in config/rps-req.pem -out config/rps-cert.pem \
+        2>> "${LOG}" || die "Failed to create RPS certificate"
+        echo "Created RPS certificate"
+    fi
+
+    rm -f config/rps-req.pem
```

Comment:
You copied the long-winded version of certificate creation which was needed only because OpenSearch requires a CA cert.  Instead, you should just copy this code: https://github.com/aryn-ai/demo-ui/blob/1ef129cf9c20b7fa7d239b40a3e98e62c1a7bab4/run-ui.sh#L30-L36